tools/migration/sender-jid-from-mam-message.escript

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env escript
+
+-mode(compile).
+-compile(nowarn_deprecated_function).
```

Comment:
why?

Reply:
I've added a comment with an explanation.

Reply:
nice, but frankly speaking I asked only because couldn't find why is that needed ;)